test/python/TestAlgebraicModels.py

```diff
@@ -1,52 +1,25 @@
 
-"""Copyright (c) 2005-2016, University of Oxford.
-All rights reserved.
-
-University of Oxford means the Chancellor, Masters and Scholars of the
-University of Oxford, having an administrative office at Wellington
-Square, Oxford OX1 2JD, UK.
-
-This file is part of Chaste.
-
-Redistribution and use in source and binary forms, with or without
-modification, are permitted provided that the following conditions are met:
- * Redistributions of source code must retain the above copyright notice,
-   this list of conditions and the following disclaimer.
- * Redistributions in binary form must reproduce the above copyright notice,
-   this list of conditions and the following disclaimer in the documentation
-   and/or other materials provided with the distribution.
- * Neither the name of the University of Oxford nor the names of its
-   contributors may be used to endorse or promote products derived from this
-   software without specific prior written permission.
-
-THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
-AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
-IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
-ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
-LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
-CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE
-GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
-HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
-LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT
-OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-"""
-
+import pytest
 import unittest
 
 import fc
 
+
 class TestAlgebraicModels(unittest.TestCase):
     """Test behaviour on models with no ODEs; based on TestClamping.hpp"""
-    def TestClampingToInitialValue(self):
-        proto = fc.Protocol('projects/FunctionalCuration/test/protocols/test_clamping1.txt')
+
+    @pytest.mark.xfail(strict=True, reason='no pycml replacement yet')
+    def testClampingToInitialValue(self):
+        proto = fc.Protocol('test/protocols/test_clamping1.txt')
         proto.SetOutputFolder('Py_TestAlgebraicModels_TestClampingToInitialValue')
-        proto.SetModel('projects/FunctionalCuration/cellml/beeler_reuter_model_1977.cellml')
+        proto.SetModel('cellml/beeler_reuter_model_1977.cellml')
```

Comment:
os.path.join

Reply:
?

Reply:
Is it not generally discouraged to write paths with slashes, and to use os.path.join instead?

(Although the only practical case where it matters is outdated macs)

Reply:
I have a vague recollection that some parts of Python at least are clever enough to translate paths written directly as strings. In any case, we're not going to be running this on Windows, and writing the paths directly looks tidier!